### PR TITLE
multi_vms_with_stress: New case to test multiple VMs with stressing

### PR DESCRIPTION
**Pull request by YongxueHong**
New a case to test multiple VMs under running stress on host.

ID: 1731466
Signed-off-by: Yongxue Hong <yhong@redhat.com>

### CODE REVIEW

File: qemu/tests/cfg/multi_vms_with_stress.cfg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+- multi_vms_with_stress:
+    virt_test_type = qemu
+    type = multi_vms_with_stress
+    start_vm = no
```
Comment (lolyu) [inline qemu/tests/cfg/multi_vms_with_stress.cfg]: should `kill_vm = yes`?

Comment (YongxueHong) [reply qemu/tests/cfg/multi_vms_with_stress.cfg]: Thanks, have updated it.

**Comment by zhenyzha** [comment]
(1/1) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.multi_vms_with_stress: PASS (790.01 s)

LGTM.Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>

**Comment by qingwangrh** [comment]
python ConfigTest.py --testcase=multi_vms_with_stress --guestname=RHEL.8.1.0 --clone=no
1/1) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.multi_vms_with_stress: PASS (352.20 s)

LGTM

**Comment by kimi1978wy** [comment]
Test failed as "ERROR: Login timeout expired" when not assigning specified cpu and mem. Log refer to the bz

**Comment by YongxueHong** [comment]
> Test failed as "ERROR: Login timeout expired" when not assigning specified cpu and mem. Log refer to the bz

Hi, @kimi1978wy 
I have updated it, pls help me to review again, thanks.

**Comment by kimi1978wy** [comment]
I saw it fixed the memory to 4G, is that ok for our test？ if feature owner accept the fix memory, it LGTM

 (1/1) Host_RHEL.m7.u8.qcow2.virtio_blk.up.virtio_net.Guest.Win8.x86_64.io-github-autotest-qemu.multi_vms_with_stress.q35: PASS (518.02 s)

Thanks
Yu Wang

**Comment by zhenyzha** [comment]
(1/1) Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.multi_vms_with_stress: PASS (868.54 s)


> I saw it fixed the memory to 4G, is that ok for our test？ if feature owner accept the fix memory, it LGTM
> 
> @qingwangrh @zhenyzha do you agree with fixing the memory=4G?
> 
> (1/1) Host_RHEL.m7.u8.qcow2.virtio_blk.up.virtio_net.Guest.Win8.x86_64.io-github-autotest-qemu.multi_vms_with_stress.q35: PASS (518.02 s)
> 
> Thanks
> Yu Wang

@kimi1978wy  @qingwangrh 
It's ok for me, and I checked this product bug and the test point is not in mem size. 4G is also used by feature owners in product bugs.
 (1/1) Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.multi_vms_with_stress: PASS (868.54 s)

LGTM.Acked-by: Zhenyu Zhang <zhenyzha@redhat.com>

**Comment by qingwangrh** [comment]
> (1/1) Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.multi_vms_with_stress: PASS (868.54 s)
> 
> > I saw it fixed the memory to 4G, is that ok for our test？ if feature owner accept the fix memory, it LGTM
> > @qingwangrh @zhenyzha do you agree with fixing the memory=4G?
> > (1/1) Host_RHEL.m7.u8.qcow2.virtio_blk.up.virtio_net.Guest.Win8.x86_64.io-github-autotest-qemu.multi_vms_with_stress.q35: PASS (518.02 s)
> > Thanks
> > Yu Wang
> 
> @kimi1978wy @qingwangrh
> It's ok for me, and I checked this product bug and the test point is not in mem size. 4G is also used by feature owners in product bugs.
> (1/1) Host_RHEL.m8.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.2.0.ppc64le.io-github-autotest-qemu.multi_vms_with_stress: PASS (868.54 s)
> 
> LGTM.Acked-by: Zhenyu Zhang [zhenyzha@redhat.com](mailto:zhenyzha@redhat.com)

If this case may run on most hosts, 4G is good for me. thanks.

**Comment by kimi1978wy** [comment]
LGTM.

Acked-by: Yu Wang <wyu@redhat.com>

**Comment by YongxueHong** [comment]
Thanks all, let's merge it.

**Pull request merged**